pkg/scheduler/framework/runtime/registry_test.go

```diff
@@ -76,7 +78,9 @@ func TestDecodeInto(t *testing.T) {
 func isRegistryEqual(registryX, registryY Registry) bool {
 	for name, pluginFactory := range registryY {
 		if val, ok := registryX[name]; ok {
-			if reflect.ValueOf(pluginFactory).Pointer() != reflect.ValueOf(val).Pointer() {
+			p1, _ := pluginFactory(nil, nil)
+			p2, _ := val(nil, nil)
```

Comment:
A nit: as `pluginFactory` is from regY, so swap the naming of p1, p2 for readability? so that p1 is mapped to the entry in regX, and p2 is mapped to regY.

```go
p1, _ := val(nil, nil)
p2, _ := pluginFactory(nil, nil)
```